src/main/java/com/ericsson/bss/cassandra/ecaudit/filter/role/RoleAuditFilter.java

```diff
@@ -71,40 +77,39 @@ public boolean isFiltered(AuditEntry logEntry)
         return isFiltered(Roles.getRoles(primaryRole), logEntry.getPermissions(), logEntry.getResource());
     }
 
+    @VisibleForTesting
```

Comment:
If unit tests are in the same package you don't need this, but I guess this is mostly added for documentation purposes?

Reply:
Yes, just for documentation and as a motivator for not making it private.